pages/usage.html

```diff
@@ -26,14 +27,13 @@
             </a>
         </header>
         <nav class="site-nav">
-            <div class="nav-heading nav-getstarted">
-                <a style="color: #027a64" href="/pages/getstarted.html"
-                    >Getting Started</a
-                >
-            </div>
-            <div class="nav-heading nav-comp-heading">
-                <a href="/pages/components/components.html">Components</a>
+            <div class="nav-main-heading">Getting Started</div>
+
+            <div class="nav-sub-heading nav-getstarted">
+                <a style="color: #027a64" href="/pages/usage.html">Usage</a>
```

Comment:
```suggestion
                <a  href="/pages/usage.html">Usage</a>
```
avoid inline style as much as you can!